Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "indicator_operatingsystemrefs",
+    "version": -1,
+    "modified": "2021-04-28T11:01:32.961855+03:00",
+    "name": "Operating System Refs",
+    "ownerOnly": false,
+    "cliName": "operatingsystemrefs",
+    "type": "tagsSelect",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "associatedToAll": true,
```

Comment:
isn't is supposed to be associated to a specific IOC?

Reply:
added

Reply:
please check that, I think it should be the details filed, which is, e.g `STIX Malware`

Reply:
This is a case where we do not have a complete list of options, so we thought (me and @ytauman) we should use `tags` (as in stix aliases).